### PR TITLE
Complement PR #34 with tests

### PR DESCRIPTION
**Pull request by jimmiehansson**
**Description**
This PR was made to complement the missing tests for #34 and to complete the integration of custom log levels passed inside the existing `opts` object. In addition, some broken tests were fixed.

### CODE REVIEW

File: multistream.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 const metadata = Symbol.for('pino.metadata')
 
-const levels = {
+var levels = {
```
Comment (jsumners) [inline multistream.js:5]: Why the revert of `const` to `var`?

File: multistream.js [hunk 2]
```diff
@@ -16,8 +16,8 @@ function multistream (streamsArray, opts) {
   var counter = 0
 
   streamsArray = streamsArray || []
-
   opts = opts || { dedupe: false }
+  levels = opts.levels || levels
```
Comment (jsumners) [inline multistream.js:20]: I don't think we should be overwriting the default set of levels. A new closure scoped `levels` should be created instead.

Comment (jimmiehansson) [reply multistream.js:20]: @jsumners I agree, good catch!

File: multistream.js (hunk 1 above)
Comment (jimmiehansson) [reply multistream.js:5]: @jsumners This was a global override, which in second-hand isn't best practice. I'll update the PR with a reference to level that is separated into `multistream` only

Comment (jimmiehansson) [reply multistream.js]: I've updated the closure, when time found please do have a look.

**Review by mcollina — COMMENTED** [review]
Can you please add some docs?

**Review by mcollina — APPROVED** [review]
lgtm

**Review by jsumners — APPROVED** [review]
LGTM

**Pull request merged**